package.json

```diff
@@ -23,9 +23,7 @@
   },
   "lint-staged": {
     "*.{js,ts,jsx,tsx}": [
-      "eslint --fix --ignore-path .gitignore"
-    ],
-    "*": [
+      "eslint --fix --ignore-path .gitignore",
```

Comment:
@gigxz This change was made because prettier was trying to format the nginx config file. It seemed to me like it should only format JS files, but please correct me if I'm mistaken about that.

Reply:
Prettier was formatting json and *.md files, which I think we should keep. I'll update to add the docker directory to the prettierignore instead.